fileio/simpleio.go

```diff
@@ -88,7 +89,32 @@ func CatchErrThrowEOF(err error) {
 // Close will return an error if it has already been called.
 func (reader *SimpleReader) Close() {
 	if reader != nil {
-		err := reader.file.Close()
-		common.ExitIfError(err)
+		common.ExitIfError(reader.close())
 	}
 }
+
+// StringToIntSlice will process a column data separated by commas, convert the slice into a slice of type int.
+func StringToIntSlice(column string) []int {
+	work := strings.Split(column, ",")
+	var answer []int = make([]int, len(work))
+	for i := 0; i < len(work)-1; i++ {
+		answer[i] = common.StringToInt(work[i])
+	}
+	return answer
+}
+
+// IntListToString will process a slice of type int as an input and return a each value separated by a comma as a string.
```

Comment:
Might be good to say it adds a trailing comma at the end?

Reply:
You got it.